Orange/widgets/model/owneuralnetwork.py

```diff
@@ -81,11 +82,11 @@ class OWNNLearner(OWBaseLearner):
     activation_index = Setting(3)
     solver_index = Setting(2)
     max_iterations = Setting(200)
-    alpha_index = Setting(0)
+    alpha_index = Setting(1)
```

Comment:
Have you considered settings migration, since alpha_index = 0 now represents alpha = 0?

Reply:
Oh boy, you are right. Thank you for noticing. Will add migration.

Reply:
I fixed it. Would you mind checking it so that it is not a complete disaster?